data/query.go

```diff
@@ -11,10 +11,14 @@ import (
 const (
 	// DefaultLimit - default values for limit query
 	DefaultLimit = 10
+	// DefaultLimitStr - default values for limit query in string format
+	DefaultLimitStr = "10"
 	// DefaultSort - default values for sort query
 	DefaultSort = "relevance"
 	// DefaultPage - default values for page query
 	DefaultPage = 1
+	// DefaultPageStr - default values for page query in string format
+	DefaultPageStr = "1"
```

Comment:
You might consider replacing this 'const' block with the following, which has the advantage of preventing anyone doing maintenance in the future from forgetting to make 'DefaultLimitStr' the equivalent of 'DefaultLimit' and similarly for 'DefaultPageStr & DefaultPage':

const (
	// DefaultLimit - default values for limit query
	DefaultLimit = 10
	// DefaultSort - default values for sort query
	DefaultSort = "relevance"
	// DefaultPage - default values for page query
	DefaultPage = 1
)

// DefaultLimitStr - default values for limit query in string format
var DefaultLimitStr string

// DefaultPageStr - default values for page query in string format
var DefaultPageStr string

func init() {
	DefaultLimitStr = strconv.Itoa(DefaultLimit)
	DefaultPageStr = strconv.Itoa(DefaultPage)
}